doc/nrf/doc_styleguide.rst

```diff
@@ -23,10 +23,13 @@ More information about RST is available in the `reStructuredText Primer`_.
 
 The |NCS| documentation follows the Zephyr |sg|, and adds a few more restrictive rules that are described below.
 
-Titles and headings
-===================
+Page structure
+==============
 
 Keep titles and headings as short and to the point as possible.
+Add table of contents if the page is long and has several sections.
```

Comment:
```suggestion
Add table of contents if the page has sections.
```
We said that we won't make any rules for the length of the page, and that we'll add the ToC even if there is only one section.